app/controllers/concerns/exception_handler.rb

```diff
@@ -20,36 +23,41 @@ def render_attributes_errors(error_messages)
     render json: { attributes_errors: error_messages }, status: :bad_request
   end
 
-  def render_standard_error(exception)
+  def render_parameter_missing(exception)
+    render_errors(I18n.t('errors.missing_param', param: exception.param.to_s), :bad_request)
+  end
+
+  def render_record_not_found(exception)
+    render_errors(I18n.t('errors.record_not_found', exception.model), :not_found)
+  end
+
+  def render_record_invalid(exception)
+    errors = exception.record.errors.messages
+    render_attributes_errors(errors)
+  end
+
+  def render_page_not_found(_error_messages)
```

Comment:
shouldn't this be `_exception`? what do you think about removing the parameter entirely from the signature since we are not using it?